packages/dev/core/src/ShadersWGSL/ShadersInclude/pbrBlockReflectivity.fx

```diff
@@ -182,6 +218,11 @@ fn reflectivityBlock(
         // In glTF's material model, this is the F0 value calculated from the IOR and then multiplied by the maximum component of the specular colour.
         outParams.reflectanceF0 = max(surfaceReflectivityColor.r, max(surfaceReflectivityColor.g, surfaceReflectivityColor.b));
         outParams.reflectanceF90 = vec3f(1.0);
+        #if (DIELECTRIC_SPECULAR_MODEL == DIELECTRIC_SPECULAR_MODEL_OPENPBR)

```

Comment:
Not sure to get the diff between reflectanceF90 and colorReflectanceF90

Reply:
`colorReflectanceF90` stores a coloured value that represents the percentage of light reflected by the specular layer. `reflectanceF90` represents the percentage of light that is reflected OR absorbed by the specular layer. For example, the color at F90 might be (0.5, 0.5, 0.0) but, assuming the weight is 1.0, the reflectance value at F90 is 1.0. So, 100% of light is absorbed or reflected by the specular lobe at F90, despite the reflected colour being a medium yellow.

Reply:
I wonder how we should name them in PBR 2 :-)